crates/mcp-macros/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "mcp-macros"
```

Comment:
would the tool macro need to be in the same cargo package where Server is defined?